The 5.1b01 release has the wrong version information set, so correct them

--- old/make/Defs.gmk
+++ new/make/Defs.gmk
@@ -445,9 +445,9 @@
 # Version tags
 #
 # BUILD_* variables are normally set (overridden) by RE builds
-BUILD_VERSION = 5.0
+BUILD_VERSION = 5.1
 BUILD_MILESTONE = dev
-BUILD_NUMBER = b00
+BUILD_NUMBER = b01
 
 # don't eval dates here directly, because that leads to unstable builds
 #BUILD_YEAR:sh = /bin/date +"%Y"
--- old/make/build.properties
+++ new/make/build.properties
@@ -65,6 +65,6 @@
 # checkstyle (should be version 5.4 or better)
 checkstyle.home = ${jtreg.build.resources}/checkstyle/5.4
 
-build.version = 5.0
+build.version = 5.1
 build.milestone = dev
-build.number = b00
+build.number = b01
